src/ImageSharp/Bootstrapper.cs

```diff
@@ -16,54 +16,48 @@ namespace ImageSharp
     /// <summary>
     /// Provides initialization code which allows extending the library.
     /// </summary>
-    public static class Bootstrapper
+    public class Bootstrapper
     {
         /// <summary>
-        /// The list of supported <see cref="IImageFormat"/>.
+        /// A lazily initialized bootstrapper default instance.
         /// </summary>
-        private static readonly List<IImageFormat> ImageFormatsList;
+        private static readonly Lazy<Bootstrapper> Lazy = new Lazy<Bootstrapper>(() => new Bootstrapper());
```

Comment:
Lazy is threadsafe, static Bootstrapper property newing up isn't see http://csharpindepth.com/Articles/General/Singleton.aspx